src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1606,6 +1606,18 @@ public function rememberToken()
         return $this->string('remember_token', 100)->nullable();
     }
 
+    /**
+     * Create a new custom column on the table.
+     *
+     * @param  string  $column
+     * @param  string  $statement
+     * @return \Illuminate\Database\Schema\ColumnDefinition
+     */
+    public function custom($column, $statement)
```

Comment:
I feel like `$table->raw()` would be a more suitable name that would also reflect the existing `DB::raw` method? Thoughts?

Reply:
Definitely open to that. Just thinking whether it wouldn't be a bit confusing, e.g. people expecting to run **any** sort of DB statement via a raw expression. But likely.. since the most common scenario is creating columns.. it could be sort-of expected that it would be used for that.

Reply:
@Jacobs63 @stevebauman 

How about calling it `rawColumn` this way we get clarity and we are explicit about the fact that it is a column?

Reply:
`rawColumn` will do.